MiniAn/minian_main.py

```diff
@@ -431,7 +432,112 @@ def load_doric_to_xarray(
 
     return varr, file_
 
-
+            
+def cross_register(minian_parameters, currentFile_AC, currentFile_A):
+    performCrossReg = minian_parameters.params_crossRegister["crossReg"]
+    if performCrossReg:
```

Comment:
The `if performCrossReg:` should be in line 65 when you call the function `cross_register()`